### PR TITLE
Pylint test integration into the pipeline for backend files

### PR DESCRIPTION
**Pull request by jamzheng109**

### CODE REVIEW

File: code/frontend/public/index.html [hunk 1]
```diff
@@ -1,14 +1,43 @@
...
+ <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Blockbuster Application</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Main Page</title>
+    <style>
```
Comment (jshilts657) [inline code/frontend/public/index.html]: Moves all styles to a css file in the assets directory

File: code/frontend/public/submitform.html [hunk 2]
```diff
@@ -1,7 +1,82 @@
+<!DOCTYPE html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Search Page</title>
+</head>
+
+<style>
```
Comment (jshilts657) [inline code/frontend/public/submitform.html]: Move all styles to a css file in assets directory

**Review by jshilts657 — CHANGES_REQUESTED** [review]
Looks good just need to moves all styles to styles file and folder.

**Review by rzheng2019 — APPROVED** [review]
Current changes look good.

**Review by acflinch — APPROVED** [review]

**Review by jshilts657 — APPROVED** [review]
Looks good

**Pull request merged**